test_logwarn

```diff
@@ -98,6 +98,7 @@ is-ignored '\[warn\].* fatal: Invalid revision range .*\.\.'
 is-ignored '\[warn\] \[pid:[0-9]*\] $'
 is-ignored '\[error\].*cmd returned 32768$'
 is-ignored '\[error\] Worker 17519 has no heartbeat'
+is-ignored '\[error\] \[pid:4360\] Publishing opensuse\.openqa\.comment\.create failed: Can\'t connect: System error \(event ID: 726555, job ID: 4926463, 9 attempts left\)'
```

Comment:
Just keep it simple:
```suggestion
is-ignored 'comment.create'
```
If we have only one line `comment.create` in in.log, we don't have to add a long string here.
Your mistake was mainly the `Can\'t` in the string. single quotes inside of single quotes are written as two singlequotes.

Reply:
It's the right thing. Rethink the logic.
You put an example line into in.log, and in the test you just want to ensure that this exact line is ignored.
Now if you pass a very complicated string to `is-ignored`, chances are good that you mistype, and the test is worthless, because then it will always be true.
You only want to ensure that this specific line is ignores, so you only add the one unique thing about this line.